lib/form/scripts.js

```diff
@@ -2,14 +2,22 @@
 
 const browserify = require('browserify');
 const stream = require('stream');
+const fs = require('fs');
+const path = require('path');
+
+const formJS = fs.readdirSync(path.join(__dirname, 'js')).map(script => {
```

Comment:
This is cool
@ayush2k this looks like it will mean your `repeatable` script will go in the ./lib/form/js directory. 
@Snugug am I reading this correctly?